Source/driver/Castro.H

```diff
@@ -41,9 +41,12 @@ enum StateType { State_Type = 0,
 #endif
 #ifdef SDC
 #ifdef REACTIONS
-		 SDC_React_Type
+		 SDC_React_Type,
 #endif
 #endif
+#ifdef REACTIONS
+                 Balance_Type,
```

Comment:
it's okay to have an extra `,` at the end?

Reply:
It seems to compile with gcc. I haven't tried with other compilers. I am hoping that it is generally OK, so that we don't have to worry about what ends up being the last item depending on what gets ifdef'd out.

Reply:
Though actually I'm about to make it defined always, so it won't matter in this case.

Reply:
okay, but if it is defined always, can we ensure that it doesn't allocate the memory unless needed?

Reply:
Indeed. We are only allocating the memory if the rebalanced_splitting option is enabled.